python_ta/cfg/visitor.py

```diff
@@ -8,20 +8,30 @@
 class CFGVisitor:
     """An astroid visitor that creates a control flow graph for a given Python module.
 
+    Initialize with an options dict to configure the visitor.
+
+    Supported Options:
+      - "separate-condition-blocks": bool
+
     Private Attributes:
     _control_boundaries: A stack of the boundaries the visitor is currently in.
         The top of the stack corresponds to the end of the list.
         (compound statement [while], {'Break'/'Continue': CFGBlock to link to})
     """
 
     cfgs: Dict[Union[nodes.FunctionDef, nodes.Module], ControlFlowGraph]
+    options: Dict[str, Any]
     _current_cfg: Optional[ControlFlowGraph]
     _current_block: Optional[CFGBlock]
     _control_boundaries: List[Tuple[nodes.NodeNG, Dict[str, CFGBlock]]]
 
-    def __init__(self) -> None:
+    def __init__(self, options: Optional[Dict[str, Any]] = None) -> None:
         super().__init__()
         self.cfgs = {}
+        if options is not None:
+            self.options = options
```

Comment:
A more robust approach would be to define a class-level "default options" dict, and then *merge it* with the passed in options dict. This will ensure the options dict always has the expected keys.